jlm/rvsdg/simple-node.hpp

```diff
@@ -120,6 +134,31 @@ SimpleNode::output(size_t index) const noexcept
   return static_cast<simple_output *>(Node::output(index));
 }
 
+struct CreateOpNodeResult
+{
+  Node * node;
+  std::vector<output *> outputs;
+};
+
+template<typename OpType, typename... OpParameters>
+CreateOpNodeResult
```

Comment:
Why does this function return the node and the results? What is the reason here? Once the caller has the node, it is no problem to also extract the results from it using the `outputs()` function. See below.

Reply:
Returning node now.